app/views/shared/_analytics_table.html.erb

```diff
@@ -103,29 +92,22 @@
             </td>
 
             <!--Follow-up patients-->
-            <% dates_for_periods(@period, 3, include_current_period: @show_current_period).each do |date| %>
+            <% dates_for_periods(@period, 6, include_current_period: @show_current_period).each do |date| %>
```

Comment:
We're replacing 9 columns with 12. Can we make sure the new layout isn't too squished on desktop _and_ mobile? PS could you add some screenshots to the PR description?

Reply:
Added. The table is horizontal scrollable so the columns don't get squished. I did add a small `min-width` to make them look a little nicer.

Reply:
Cool. Can we run this by the design team?